noaa-cpfp-point/README.md

```diff
@@ -0,0 +1,24 @@
+# Setting up the NOAA CPFP POINT
+
+### Steps
+* Install Node
+* Install NVM (optional).
+    * If NVM installed, `nvm use`
+    * Else, upgrade to latest LTS version. (atleast v.16.17.0)
+* Install dependencies using `yarn` or `npm i`
+* create `.env` file in the proect root. A copy of `.env.example` with your own mapbox access token.
+    - Note: To run the application locally, put \
+        `PUBLIC_URL=http://localhost:<port>`
+    - Example: \
+        `PUBLIC_URL=http://localhost:8080`
+* `yarn start` to start the project.
+
+# Building the NOAA CPFP POINT
+
+### Steps
+
+In the console do the following:
+* `export PUBLIC_URL=XXX`, as an env variable
+* `yarn development` for development build
+* `yarn production` for production build
```

Comment:
do we need to update the cicd pipeline to use this command instead?

Reply:
do you mean using NVM and Yarn on CICD?

Reply:
`PUBLIC_URL` is already available in env in CICD
![image](https://github.com/user-attachments/assets/0fec7e35-4887-4464-951b-72beb4f81c88)

Reply:
just asking if we need, if we don't and the existing command works , it's fine :)

Reply:
Existing commands would work fine. But changed it to use Node version from `nvmrc`, and use `yarn` package manger instead of `npm`.

Reply:
![image](https://github.com/user-attachments/assets/539ccfef-f44e-41df-bc48-ca7bd23b6970)